### DISCUSSION TITLE
描边占位

### DISCUSSION THREAD

**Discussion by lzy280948644** [ant-design/ant-design#48120] [Q&A]
![1711526471502](https://github.com/ant-design/ant-design/assets/66244335/07996385-6626-4fd7-8ea2-f6a66c7fd551)

为什么前端，描边都喜欢把填充加上这一个像素的描边，不加或者不去算这一个像素的占位不会更好统一管理和维护吗，既然加了有的地方又不加这是为什么，不理解，请大佬们指教一二

**Comment by arvinxx** [comment]
你自己试试看就知道了，不占位勾选的时候描边会增加实际的宽高，会导致页面抖动。

**Comment by lzy280948644** [comment]
因为我是UI设计，所以不太懂代码的逻辑麻烦再请教下
1、如果说固定宽高不要描边也是会照成抖动？
2、只要有描边就要算它占位是吗，两个状态之间也会照成抖动？

&nbsp;


power
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Arvin ***@***.***&gt;; 
发送时间: 2024年3月27日(星期三) 下午4:05
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [ant-design/ant-design] 描边占位 (Discussion #48120)





 
你自己试试看就知道了，不占位勾选的时候描边会增加实际的宽高，会导致页面抖动。
